va/va.go

```diff
@@ -534,6 +556,80 @@ func (va VAImpl) fetchHTTP(identifier string, token string) ([]byte, string, *ac
 	return body, url.String(), nil
 }
 
+func (va VAImpl) getTXTEntry(name string) ([]string, error) {
+	ctx, cancelfunc := context.WithTimeout(context.Background(), validationTimeout)
+	defer cancelfunc()
+
+	if va.customResolverAddr == "" {
+		return net.DefaultResolver.LookupTXT(ctx, name)
+	}
+
+	txts := []string{}
+	client := new(dns.Client)
```

Comment:
I think it would make sense to hoist the `*dns.Client` to a field on the `VAImpl` that may be `nil` (in the case where `va.customResolverAddr` isn't in use). That way it could be constructed once instead of per-lookup.

Reply:
Done.